src/App.tsx

```diff
@@ -1,8 +1,35 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import './App.scss';
+import 'bulma';
+import { Header } from './components/Header';
+import { PeoplePage } from './components/PeoplePage';
 
-const App = () => (
+const App: React.FC = () => (
   <div className="App">
-    <h1>People table</h1>
+    <div className="container">
+
+      <Header />
+
+      <Routes>
```

Comment:
You forgot about Navigate comp.[ Checklist 4](https://github.com/mate-academy/react_people-table-basics)